rails-snow-queen/app/assets/javascripts/quotingPanel.js

```diff
@@ -0,0 +1,49 @@
+class QuotingPanel {
+  constructor() {
+    this.polygons = {};
+    this.totalAreaInSqFt = null;
+    this.totalDue = null;
+  }
+
+  showAddress(geocodedAdress) {
+    if (geocodedAdress) {
+      let addressNode = document.getElementById("displayAddress");
+      while (addressNode.firstChild) { addressNode.removeChild(addressNode.firstChild) }
+      let splitAddress = geocodedAdress.split(",");
+      splitAddress.forEach(element => {
+        let div = document.createElement("div");
+        div.innerText = element;
+        addressNode.appendChild(div);
+      });
+      addressNode.classList.remove("hidden");
+    }
+  }
+
+  updateTotals(polygons) {
+    let SQ_FT_CONVERT = 10.764;
+    const PRICE_PER_SQ_FT = 0.15;
+
+    let totalAreaInMts = 0;
+    this.polygons = polygons;
+    console.log("this.polygons: ", this.polygons)
```

Comment:
When committing code, you should try to get into the habit of not committing console logs. I know that right now they are useful for testing but it's a bad habit that might lead to `debugger` and such being introduced in error 😿 

Here is a great little snippet I like you use in VS Code to make created logs much quicker. 
```javascript
{
    "Print arg to console": {
        "prefix": "clog",
        "body": [
            "console.log(\"$1: \", $1);"
        ],
        "description": "Log output to console"
    },
}
```

You install it by going to the snippet menu (`code => preferences => User Snippets`) and replace the `javascript.json` contents with the above code snippet.

Reply:
Yes i keep forgetting to delete the console logs.

Reply:
Done